Breeder/BR_Util.cpp

```diff
@@ -2233,6 +2236,28 @@ int GetTrackEnvHeight (TrackEnvelope* envelope, int* offsetY, bool drawableRange
 	return envelopeHeight;
 }
 
+int GetTrackSpacerHeight (MediaTrack* track)
+{
+	// maybe cache the app version in a global var to avoid an API call here
+	// and in other places where we perform this check
+    bool trackSpacerIsSupported = atof(GetAppVersion()) >= 7 ? true : false;
```

Comment:
That ternary is redundant as `>=` already gives the desired boolean so it's redundant.

Reply:
I think I got it I think